locales/en-US/app.l20n

```diff
@@ -53,8 +53,8 @@
 <experimentListPageHeader "Ready for Takeoff!">
 <experimentListPageSubHeader  "Pick the features you want to try. <br> Check back soon for more experiments.">
 
-<experimentListInactiveHover "Get Started">
-<experimentListActiveHover "Manage">
+<experimentListLaunched "Just Launched">
```

Comment:
we can get rid of this line till the experiment modified time is added (we can file a followup bug for that)